sidecar/src/figwheel_sidecar/repl.clj

```diff
@@ -83,7 +83,10 @@
                        (first (string/split x #"\?")))))
        stack-trace))
 
-(defrecord FigwheelEnv [figwheel-server]
+(defrecord FigwheelEnv [figwheel-server repl-opts]
+  cljs.repl/IReplEnvOptions
```

Comment:
It's important to know what version of CLJS introduced this IReplEnvOptions.  Is this going to cause legacy nightmares or has it been around long enough that this isn't a problem.